### DISCUSSION TITLE
Can Remix dev server be configured to start in https?

### DISCUSSION THREAD

**Discussion by troutgirl** [remix-run/remix#3568] [Proposals]
Hello, I prefer to do all my development in https to make testing easier and to keep everything in sync with production. Is it possible to configure Remix dev server to take a cert and run in https mode? Also is there a way to manually specify an IP address? Thank you!

**Comment by benwis** [comment]
Hi @troutgirl! You should have complete access to the express initialization in the `server.js` in the root of the Express based templates/stacks, so I see no reason why you couldn't change IP address, port, or start an HTTPS server for those at least. But it's not something I've tried before.

StackOverflow says it should be very easy, as long as you have a cert!
https://stackoverflow.com/questions/11744975/enabling-https-on-express-js

**Reply by kiliman** [reply]
I have a script that will eject from RAS to Express.

https://rmx.fyi/ejectras

**Reply by benwis** [reply]
If I understand your script correctly @kiliman, it's a one time thing right? Once you eject, you pretty much have the express starter?

**Reply by machour** [reply]
Yes @benwis, this is what that script is all about

**Reply by kiliman** [reply]
Yeah, it pulls the latest Express template and basically updates your project and *package.json* with the differences.
